connectors/webhook-connector/element-templates/webhook-connector-intermediate.json

```diff
@@ -0,0 +1,207 @@
+{
+  "$schema": "https://unpkg.com/@camunda/zeebe-element-templates-json-schema/resources/schema.json",
+  "name": "Webhook Connector",
+  "id": "io.camunda.connectors.webhook.WebhookConnectorIntermediate.v1",
+  "version": 1,
+  "description": "Configure webhook to receive callbacks",
+  "documentationRef": "https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/http-webhook/",
```

Comment:
Will we have still this docs link or will it be another one?

Reply:
I think we should stick to the same one